views/templates/hook/search-top.tpl

```diff
@@ -23,17 +23,6 @@
 * SOFTWARE.
 *}
 
-{*<div id="search_block_top" class="col-sm-4 clearfix">
```

Comment:
Is it on purpose you have comment out this section ?

Reply:
https://github.com/clerkio/clerk-prestashop/blob/master/views/templates/hook/search-top.tpl#L26-L36

was already like this, in my opinion we could just delete it

Reply:
Delete it, and just clean up the files for out commented code, then after that, we are ready for release :)

Reply:
Yes sir, I shall do the needful